### PR TITLE
Refactor:

### PR DESCRIPTION
**Pull request by Chakravarthi0**
Refactor: Refactored navbar
Refactor: Refactored hero section
Style: Changed fonts
Style: minor style changes
Feat: Added scroll to top
Feat: Added browse by category
Feat: Added login as guest

preview: https://deploy-preview-8--the-emporium.netlify.app/

### CODE REVIEW

File: src/backend/db/users.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const users = [
     firstName: "John",
     lastName: "Doe",
     email: "johndoe@gmail.com",
-    password: "12345678",
+    password: 12345678,
```
Comment (msdeshmukh009) [inline src/backend/db/users.js:33]: Password should be string, right?

File: public/index.html [hunk 2]
```diff
@@ -10,6 +10,9 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300&family=Roboto:wght@300&family=Yellowtail&display=swap" rel="stylesheet">
```
Comment (msdeshmukh009) [inline public/index.html:15]: Also, add your favicon as well.

File: src/layouts/navbar/AsideNavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  wishlistItems,
+  cartItems,
+  setIsNavOpen,
+}) {
+  return (
+    <div className="aside-navbar">
+      {/* <div className="aside-nav-search-container aside-nav-search icon">
+        <i className="fa fa-search" aria-hidden="true"></i>
+        <input className="nav-search-input" placeholder="Search" />
+      </div> */}
```
Comment (msdeshmukh009) [inline src/layouts/navbar/AsideNavBar.jsx]: Remove this if not needed.

File: src/layouts/navbar/navbar.css [hunk 4]
```diff
@@ -1,34 +1,122 @@
...
   border: none;
 }
 @media screen and (max-width: 800px) {
-  .nav-links-container {
-    display: flex;
-    flex-direction: column;
+  .nav-links-container,
+  .nav-search {
+    /* display: flex;
+    flex-direction: column; */
```
Comment (msdeshmukh009) [inline src/layouts/navbar/navbar.css]: comments here

**Review by msdeshmukh009 — APPROVED** [review]
The UI is looking good. Nice work!!

**Review by manojkumar8400 — APPROVED** [review]

**Review by preethasri — APPROVED** [review]
good to merge

File: src/backend/db/users.js (hunk 1 above)
Comment (Chakravarthi0) [reply src/backend/db/users.js:33]: Hey Mahesh, using string gave me a 404 error. It started working fine when I changed it to be a number.

**Pull request merged**